msg.go

```diff
@@ -52,46 +52,35 @@ const (
 
 // Msg is our interface to represent an incoming or outgoing message
 type Msg interface {
+	Event
+
 	ID() MsgID
 	UUID() MsgUUID
+	ExternalID() string
 	Text() string
 	Attachments() []string
-	Locale() i18n.Locale
-	ExternalID() string
 	URN() urns.URN
-	URNAuth() string
-	URNAuthTokens() map[string]string
-	ContactName() string
+	Channel() Channel
+
+	// outgoing specific
```

Comment:
might be even cleaner to have separate interfaces for in and out...

Reply:
I will try to make that change